gordon/router.py

```diff
@@ -103,8 +103,8 @@ def _get_next_phase(self, event_msg):
         event_msg.update_phase(next_phase)
         next_plugin = self.phase_plugin_map.get(next_phase)
         if next_phase in self.FINAL_PHASES and not next_plugin:
-            msg = (f'Dropping message"{event_msg}", final phase "{next_phase}"'
-                   'not implemented.')
+            msg = (f'Dropping message"{event_msg.msg_id}", final phase '
+                   f'"{next_phase}" not implemented.')
```

Comment:
When adding this line in an earlier commit, I noticed that what we were outputting might not be that useful, and we hadn't required that EventMessages have the phase attribute that the router uses either, which could cause Gordon to crash, so I added this commit.

Reply:
Nice catch! Would you mind adding a space between `message` and `"{event_msg.msg_id}"`? That was my mistake originally, didn't even notice it before.

(sidebar topic) What I think I intended when actually writing this string message was to define a `__repr__` method (which would get implicitly called here) that would go something like

```python
# implements IEventMessage
class SomeEventMessage:
    # snip
    def __repr__(self):
        return f'<SomeEventMessage(id={self.msg_id})>'
```

and therefore would look like:

```
'Dropping message "<SomeEventMessage(id=1234)>"...'
```

I'm not making this suggestion for here; I just would like to start using `__repr__`s more to our advantage, since we can stuff it with as much (or as little) as we want, and just need to use `{<instance_name>}` rather than `{<instance_name.some_attribute}`.